form/sendmail.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+//https://www.qservers.net/process/knowledgebase.php?action=displayarticle&id=52
+// ini_set("include_path", '/home/ballers7/php:' . ini_get("include_path") );
```

Comment:
Tiny Nit: Not sure why this commented out. If a line of code is not used, it is better to remove it instead of commenting it out.

Reply:
its a php server path setting. works in prod but not locally